packages/app/obojobo-express/express_lti_launch.js

```diff
@@ -125,22 +125,13 @@ exports.assignment = (req, res, next) => {
 		return Promise.resolve()
 	}
 
-	// allows launches to redirect /view/example to /view/00000000-0000-0000-0000-000000000000
-	// the actual redirect happens in the route, this just handles the lti launch
-	let currentUser = null
-	let currentDocument = null
-
 	return Promise.resolve(req.lti)
 		.then(lti => userFromLaunch(req, lti.body))
-		.then(launchUser => {
-			currentUser = launchUser
-			return req.requireCurrentDocument()
-		})
-		.then(draftDocument => {
-			currentDocument = draftDocument
+		.then(() => req.requireCurrentDocument())
```

Comment:
Are we able to do this?

```suggestion
		.then(req.requireCurrentDocument)
```

Reply:
I struggle with this - Here's my thoughts on a conventsion: if the function being called doesn't use the return from the previous promise, then it should isolate the method like in the PR.

Reply:
I can get behind that